server/enterprise/message_export/csv_export/csv_export.go

```diff
@@ -404,27 +401,13 @@ func postToRow(post *model.MessageExport, createTime *int64, message string) []s
 }
 
 func attachmentToRow(post *model.MessageExport, attachment *model.FileInfo) []string {
-	row := postToRow(post, post.PostCreateAt, *post.PostMessage)
-
-	attachmentEntry := fmt.Sprintf("%s (files/%s/%s-%s)", attachment.Name, *post.PostId, attachment.Id, path.Base(attachment.Path))
-	attachmentMessage := "attachment"
-	userType := row[len(row)-2]
-
-	if attachment.DeleteAt > 0 && post.PostDeleteAt != nil {
```

Comment:
A little bug, yep. Assuming `post.PostDeleteAt` was definitive.  (It's still there, just moved, but it's being fixed in the WIP PR, moving CSV to use the generalized data).  Now, whether or not that would  cause a file to be mistakenly be labelled deleted? I'm not sure... Maybe if a user added the `DeleteAt` to a post through the API? Otherwise, I don't think it would (the use case for DeleteAt without the props is to signal an edited version of a post, iirc).